webpack.common.js

```diff
@@ -57,9 +55,8 @@ module.exports = function (mode) {
                                 /* Remove any debugger statements for a production build */
                                 pattern: /debugger;|debugger/g,
                                 replacement: function (match, p1, offset, string) {
-                                    // console.log('####################### STRING REPLACE PLUGIN >> match: ' + match + ' --- p1: ' + p1 + /* ' --- offset: ' + offset + */' --- string: ' + string);
                                     if (mode === buildProps.MODE_PRODUCTION) {
-                                        throw new Error('*** ERROR: HEY YOU! REMOVE THE debugger; STATEMENT FROM YOUR CODE! ***');
+                                        return '';
```

Comment:
I actually put that feedback in there on purpose so that it would be impossible to do a production build with a debugger statement in there. Can we leave it?